spatial_navigation.js

```diff
@@ -1062,6 +1062,7 @@
         _sections[sectionId] = undefined;
         _sections = extend({}, _sections);
         _sectionCount--;
+        _lastSectionId === sectionId && (_lastSectionId = '');
```

Comment:
Excellent! Thanks a lot for catching this.

For the consistency of the coding style, would you mind changing to `if` block?
```js
if (_lastSectionId === sectionId) {
  _lastSectionId = '';
}
```

Reply:
Yes, I'll change this. 

Quick question: can you publish this lib on npm? I can provide package.json and appropriate changes to do that.

Reply:
Although I'm not good at npm, I think I can try.